apps/meteor/tests/e2e/omnichannel/omnichannel-livechat-api.spec.ts

```diff
@@ -224,11 +226,12 @@ test.describe('OC - Livechat API', () => {
 			agent = await createAgent(api, 'user1');
 			agent2 = await createAgent(api, 'user2');
 
-			departments = await Promise.all([createDepartment(api), createDepartment(api)]);
-			const [departmentA, departmentB] = departments.map(({ data }) => data);
+			departments = await Promise.all([createDepartment(api), createDepartment(api), createDepartment(api, { showOnRegistration: true })]);
+			const [departmentA, departmentB, departmentC] = departments.map(({ data }) => data);
```

Comment:
Can we delete these `after` all?

Reply:
We already do in https://github.com/RocketChat/Rocket.Chat/pull/32498/files#diff-316864adc76907c57aa42a0329a138a24eb997901583e397a4b328321c54d611R273